src/libsystemd/sd-journal/journal-file.c

```diff
@@ -1972,7 +1972,7 @@ static int journal_file_append_entry_internal(
                 const dual_timestamp *ts,
                 const sd_id128_t *boot_id,
                 uint64_t xor_hash,
-                const EntryItem items[], unsigned n_items,
+                const EntryItem *items, unsigned n_items,
```

Comment:
why this change btw? we use the syntax sometimes to indicate that we expect more than a single entry. it's entirely equivalent otherwise, so I think it's a good hint.

Reply:
Ahhh, I thought we always used pointer syntax for arrays, will drop this commit then.

Reply:
i mean for char arrays we use the pointer syntax. We are not systematic in our use...

But I think in general we should prefer [] syntax for parameters that are array like if the type they are of is not a trivial type or so. but hte lines are blurry.

I mean if you see a * argument it can be one param or more, noone knows, but if you see [] then I think it's clear that it can be multiple.